source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,43 @@
+---
+title: Direct Debit 
+weight: 75
+---
+
+# GOV.UK Pay Variable Direct Debit
+
+Before you read this documentation, you should be familiar with how the
+[GOV.UK Pay API enables users to make
+payments](/payment_flow_overview/#payment-flow). Your contract with GOV.UK Pay
+should also cover Direct Debit, and you should have a merchant account with
+GoCardless. 
+
+If you have already integrated with the GOV.UK Pay API, you can start using 
+Direct Debit quickly. You should: 
+
+* request a GOV.UK Pay Direct Debit test account [from our 
+support team](/support_contact_and_more_information/#contact-us).
+* add support for the Direct Debit endpoints to your existing GOV.UK Pay integration. 
+* sign in to your GOV.UK Pay Direct Debit account and connect your GOV.UK Pay
```

Comment:
when we revisit dev on DD from Jan 2019, we might look at improving the existing OAuth flow, so instead of connecting a pre-existing GC account, you set one up via the OAuth flow initiated from Pay. So can we be a little less specific on here? Maybe something like "sign in...and use our OAuth flow to setup an account with our Direct Debit supplier." Ugh that's maybe too specific the other way... although i'm pretty sure that setting up the account via OAuth is the product direction we'll aim to go in!!

Reply:
OK: is there a need to get these docs out before Jan 19? I'm happy to just put whatever you like in here but if it's going to be extensively revisited before that then perhaps we can just manage with the Apiary docs until then?

Reply:
yes, we want to start showing services that we have a real DD offering. they can start integrating with us before we're ready to deliver the DD service so having API reference is useful. would prefer it to be in our tech docs rather than Apiary as it all links up better

Reply:
OK, no problem